css/styles.css

```diff
@@ -524,12 +566,25 @@ textarea {
   }
 
   .dottedDeco {
-    visibility: visible;
+    display: inline-block;
+    width: 213px;
+    position: absolute;
+    top: 766px;
+    right: 663px;
+  }
+
+  .dottedDeco2 {
+    display: inline-block;
+    width: 213px;
+    position: absolute;
+    top: 1111px;
+    right: 933px;
   }
 
   .aboutSection {
     display: flex;
     flex-direction: row-reverse;
+    margin-top: 1800px;
   }
```

Comment:
Kindly adjust the about section in order to match that given by the design template.

Your design:
![image](https://user-images.githubusercontent.com/20554583/200152657-de2c5f06-6e6c-44f6-b62e-ca0a5e052653.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/200152584-63c8128d-a4e0-4c8b-8585-25a8f61844d8.png)